model/loss/base_loss.py

```diff
@@ -557,3 +557,57 @@ def call(self, y_true: Dict[Text, tf.Tensor],
         weights)
     # Reduce_sum over the channels (i.e., number of masks).
     return tf.reduce_sum(weighted_dice_losses, axis=-1)
+
+
+class DepthLoss(tf.keras.losses.Loss):
+  """This class contains code to compute the depth loss.
+
+  This depth loss function combines the scale invariant logarithmic error and
```

Comment:
We should rename the class to better reflect that (it combines the scale invariant logarithmic error and relative squared error). Also, should we have two classes (one for each)?

Please add reference paper for those losses.